subsys/bluetooth/host/cs.c

```diff
@@ -561,6 +856,13 @@ void bt_hci_le_cs_test_end_complete(struct net_buf *buf)
 		return;
 	}
 
+	struct net_buf *reassembly_buf = get_reassembly_buf(BT_HCI_LE_CS_TEST_CONN_HANDLE, false);
+
+	if (reassembly_buf) {
+		LOG_WRN("De-allocating a dangling reassembly buffer");
```

Comment:
Will it also be left dangling on a disconnect? Maybe need to add a disconnected callback to handle this?